static/src/javascripts/projects/commercial/modules/third-party-tags/twitter-uwt.spec.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import { twitterUwt } from 'commercial/modules/third-party-tags/twitter-uwt';
+import { getAdConsentState as _getAdConsentState } from 'common/modules/commercial/ad-prefs.lib';
+
+const getAdConsentState: any = _getAdConsentState;
+
+jest.mock('common/modules/commercial/ad-prefs.lib', () => ({
+    getAdConsentState: jest.fn(),
+}));
+
+/**
+ * we have to mock config like this because
```

Comment:
Since twitterUwt returns a function and not an object like most third party tag modules we can test the 'regular' way.